x-pack/plugins/enterprise_search/public/applications/workplace_search/views/security/security.test.tsx

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import '../../../__mocks__/kea.mock';
```

Comment:
I may br wrong but I don't think we need the top import anymore, thanks to some refactoring by the App Search team